contrib/QtCreator/MEGAclc/MEGAclc.pro

```diff
@@ -1,58 +1,23 @@
-TEMPLATE = app
 CONFIG += console c++11
 CONFIG -= app_bundle
 CONFIG -= qt
-
-TARGET = megaclc
-
-debug_and_release {
-    CONFIG -= debug_and_release
-    CONFIG += debug_and_release
-}
-CONFIG(debug, debug|release) {
-    CONFIG -= debug release
-    CONFIG += debug
-}
-CONFIG(release, debug|release) {
-    CONFIG -= debug release
-    CONFIG += release
-}
-
-DEFINES += LOG_TO_LOGGER
-
-CONFIG += USE_LIBUV
-CONFIG += USE_MEGAAPI
-CONFIG += USE_MEDIAINFO
-CONFIG += ENABLE_CHAT
-CONFIG += USE_WEBRTC
 CONFIG += USE_AUTOCOMPLETE
 CONFIG += USE_CONSOLE
-
-include(../../../bindings/qt/megachat.pri)
-
+CONFIG += MEGACLC
+TARGET = megaclc
+LIBS += -lstdc++fs -lreadline -ltermcap
 DEPENDPATH += ../../../examples/megaclc
 INCLUDEPATH += ../../../examples/megaclc
-
+INCLUDEPATH += ../../../third-party/mega/sdk_build/install/include
+QMAKE_LIBDIR += ../../../third-party/mega/sdk_build/install/lib
 SOURCES +=  ../../../examples/megaclc/megaclc.cpp
 
-win32 {
-    QMAKE_LFLAGS += /LARGEADDRESSAWARE
-    QMAKE_LFLAGS_WINDOWS += /SUBSYSTEM:WINDOWS,5.01
-    QMAKE_LFLAGS_CONSOLE += /SUBSYSTEM:CONSOLE,5.01
-    DEFINES += PSAPI_VERSION=1
-}
+include(../../../bindings/qt/megachat.pri)
 
 macx {
```

Comment:
Is not this block part of the new `megachat.pri`, @jgandres?

Reply:
Removed